### PR TITLE
Add support for ROS2 Galactic and Rolling

### PR DESCRIPTION
**Pull request by TSC21**
Some minor changes so to have the package working with ROS 2 Galactic and Rolling.

### CODE REVIEW

**Comment by TSC21** [comment]
Pending the `librealsense2` release with bloom: https://github.com/IntelRealSense/librealsense2-release/issues/4 (which is causing the CI failure for Galactic and Rolling).

**Comment by TSC21** [comment]
@doronhi anything blocking this from being merged? The change is quite trivial and should unlock others working with the latest distros.

File: .travis.yml [hunk 1]
```diff
@@ -2,13 +2,19 @@ sudo: required
 matrix:
     include:
         - dist: bionic
+          env: _ros_dist=dashing
         - dist: focal
+          env: _ros_dist=foxy
+        - dist: focal
+          env: _ros_dist=galactic
+        - dist: focal
+          env: _ros_dist=rolling
```
Comment (doronhi) [inline .travis.yml]: Please remove rolling from the travis's tests as there are no plans to distribute librealsense2 debian package for that release.

**Review by doronhi — CHANGES_REQUESTED** [review]

**Comment by doronhi** [comment]
There are currently 2 other PRs on that matter: #1897, #1892
I think I'll choose this one as it includes the .travis.yml modifications. I don't want the Rolling to be in the .travis as this is a version we have no plans to distribute a Debian for.

File: .travis.yml (hunk 1 above)
Comment (TSC21) [reply .travis.yml]: I will remove it. Thanks!

**Comment by TSC21** [comment]
> There are currently 2 other PRs on that matter: #1897, #1892
> I think I'll choose this one as it includes the .travis.yml modifications. I don't want the Rolling to be in the .travis as this is a version we have no plans to distribute a Debian for.

Oh I didn't notice it. Glad I can contribute though with this and the Travis CI extension.

File: .travis.yml (hunk 1 above)
Comment (TSC21) [reply .travis.yml]: @doronhi done!

**Comment by doronhi** [comment]
I have no more comments. Just waiting with the merge until I manage to add the librealsense2 debian for galactic so that the travis runs will not fail.

**Pull request merged**